html-css/week1/Nina/portfolio/style.css

```
@@ -0,0 +1,376 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+
+  font-family: "Open Sans", sans-serif;
+  font-style: normal;
+  display: flex;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: black
+}
+
+a:hover {
+
+  text-decoration: underline;
+
+}
+
+.container {
+  max-width: 1440px;
+  margin: auto;
+  gap: 30px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+
+  @media(max-width:1440px) {
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+}
+
+nav {
+  max-width: 1440px;
+  width: 100%;
+  background-color: rgba(212, 236, 255, 0.623);
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+
+  ul {
+    display: flex;
+    align-items: Center;
+    list-style-image: none;
+
+
+    li {
+      display: inline-block;
+      margin-right: 10px;
+
+      a {
+        display: inline-block;
+        padding: 10px 20px;
+
+        &:hover {
+          background-color: rgba(241, 240, 240, 0.962);
+          border-radius: 10px;
+          text-decoration: none;
+        }
+
+      }
+    }
+  }
+
+  @media(max-width:768px) {
+    nav {
+      width: 100%;
+      margin-top: 10px;
+    }
+
+    .contact-me-button {
+      height: auto;
+    }
+  }
+}
+
+.button {
+  background-color: #0a61ae;
+  border-radius: 10px;
+  color: white;
+  height: 40px;
+
+  &:hover {
+    background-color: black;
+    transition: ease 0.3s all;
+  }
+
+
+}
+
+.hero {
+  display: flex;
+  max-width: 1440px;
+  gap: 20px;
+
+  @media(max-width:1024px) {
+    flex-direction: column;
+  }
+
+  margin-top: 30px;
+
+  .hero-title {
+    color: rgba(39, 102, 237, 0.719);
+    font-size: 45px;
+  }
+
+  .hero-h1 {
+    font-size: 40px;
+    line-height: 1;
+    margin-bottom: 10px;
+
+    small {
+      display: block;
+      font-weight: 50;
+    }
+  }
+
+  .hero-blue {
+    flex: 1;
+    background-color: rgba(212, 236, 255, 0.623);
+    border-radius: 30px;
+    padding: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .call-to-action {
+      margin-top: 20px;
+      margin-bottom: 10px;
+
+      a {
+        margin-right: 10px;
+        margin-bottom: 10px;
+
+      }
+    }
+
+
+  }
+
+
+  .hero-pink {
+    flex: 1;
+    background-color: #f7906868;
+    border-radius: 30px;
+    padding: 0px 30px;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+
+    img {
+      margin-top: -60px;
+      max-width: 420px;
+      margin-bottom: 0;
+    }
+
+  }
+
+  .call-to-action {
+    a {
+      padding: 10px;
+    }
+  }
```

Instead of extra nesting you could write the selector directly as this:
```suggestion
  .call-to-action a {
      padding: 10px;
  }
```
but even further better, you could create a new class `.hero__link` and avoid the lengthy and specific selector of `.hero  & .hero-blue & .call-to-action & a`